net-misc/openssh/files/sshd.service.1

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=OpenSSH server daemon
+After=syslog.target network.target auditd.service
+
+[Service]
+ExecStartPre=/usr/bin/ssh-keygen -A
+ExecStart=/usr/sbin/sshd -D -e
+ExecReload=/bin/kill -HUP $MAINPID
+KillMode=process
+OOMPolicy=continue
+Restart=on-failure
+RestartSec=42s
+Type=notify
```

Comment:
OpenSSH does not support Type=notify. 
You probably took this from a distro that patches OpenSSH to support it. Gentoo does not apply any such patch.
Please drop this setting.

Reply:
I thought I did that once I established that we weren't applying that patch!